prow/sidecar/run.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 
+	pkgio "k8s.io/test-infra/pkg/io"
```

Comment:
Intent is that doing this should involve deleting the "io" import not having both existing side by side where one is renamed (but I suspect this should really just be the gcs import)